pages/docs/prompts/get-started.mdx

```diff
@@ -203,6 +257,16 @@ langfuse_prompt = langfuse.get_prompt("movie-critic")
 langchain_prompt = ChatPromptTemplate.from_template(langfuse_prompt.get_langchain_prompt())
 ```
 
+Chat prompts
+
+```python
+# Get current production version of a chat prompt
+langfuse_prompt = langfuse.get_prompt("movie-critic-chat", type="chat")
+
+# Create a Langchain ChatPromptTemplate from the Langfuse prompt chat messages
+langchain_prompt = langfuse_prompt.get_langchain_prompt()
```

Comment:
Potential version/import issues when returning langchain `ChatPromptTemplate` here or ok? I assume ok since optional dependencies are handled well in python